Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts

```diff
@@ -50,5 +51,5 @@ app.use((err, req, res, next) => { // eslint-disable-line @typescript-eslint/no-
 app.set('port', process.env.PORT || 3000);
 
 const server = app.listen(app.get('port'), function () {
-    debug('Express server listening on port ' + server.address().port);
+    debug('Express server listening on port ' + (server.address() as AddressInfo).port);
```

Comment:
How do we know it's not a `string`?

Reply:
[server.address()](https://nodejs.org/api/net.html#net_server_address) will only return a string if the server is listening on a pipe or Unix domain socket, which (at least in the unmodified template) will never be true.